examples/drivers/windows/otLwf/filter.h

```diff
@@ -144,7 +144,7 @@ typedef struct _MS_FILTER
     IN6_ADDR                    otLinkLocalAddr;
     otNetifAddress              otAutoAddresses[OPENTHREAD_CONFIG_NUM_SLAAC_ADDRESSES];
 #if OPENTHREAD_ENABLE_DHCP6_CLIENT
-    otNetifAddress              otDhcpAddresses[OPENTHREAD_CONFIG_NUM_DHCP_PREFIXES];
+    otDhcpAddress               otDhcpAddresses[OPENTHREAD_CONFIG_NUM_DHCP_PREFIXES];
```

Comment:
Not quite. The Windows structure being referenced here is a [MIB_UNICASTIPADDRESS_ROW](https://msdn.microsoft.com/en-us/library/windows/desktop/aa814507(v=vs.85).aspx). For the time being we will just have to map between the `bool` and `0xffffffff`.